_posts/2017-02-02-Rust-1.15.md

```diff
@@ -36,13 +36,13 @@ struct Pet {
 The `Debug` trait is then implemented for `Pet`. However, this only worked for
 traits provided as part of the standard library; it was not customizable. With
 Rust 1.15, it now is. That means, if you want to turn your `Pet` into JSON, it's
-as easy as adding [serde] to your `Cargo.toml`:
+as easy as adding [Serde][serde] to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-serde = "0.9.0"
-serde_derive = "0.9.0"
-serde_json = "0.9.0"
+serde = "0.9"
```

Comment:
The two are equivalent so why write more I guess. To me this refers to "the entire 0.9 line" without needing to think about patch versions. http://doc.crates.io/specifying-dependencies.html uses this 4 times and that 4 times.